plugin/pkg/scheduler/algorithm/types.go

```diff
@@ -18,35 +18,13 @@ package algorithm
 
 import (
 	"k8s.io/kubernetes/pkg/api"
+	schedulerapi "k8s.io/kubernetes/plugin/pkg/scheduler/api"
 )
 
 // FitPredicate is a function that indicates if a pod fits into an existing node.
 type FitPredicate func(pod *api.Pod, existingPods []*api.Pod, node string) (bool, error)
 
-// HostPriority represents the priority of scheduling to a particular host, lower priority is better.
```

Comment:
A lot of diffs are caused by the move-around of these API types.
Can we separate these changes to another PR? This can help make the review clearer here :)